app/views/News.js

```diff
@@ -85,7 +86,7 @@ class NewsScreen extends Component {
 
     GetStoreData(AUTHORITY_NEWS)
       .then(nameNewsString => {
-        // Bring in news from the various authorities.  This is
+3        // Bring in news from the various authorities.  This is
```

Comment:
```suggestion
        // Bring in news from the various authorities.  This is
```

Reply:
No need this. will remove and check in again.